.storybook/preview.js

```diff
@@ -14,30 +13,39 @@ const centerStyle = {
   justifyContent: 'center',
   alignItems: 'center',
   height: '100%',
-  padding: '15rem'
+  padding: '3rem'
```

Comment:
Question, I know internally we try to stick to pixels. What is the reason we are using rem here?

Reply:
Back in the day everything was in REM and I'm not really sure why. Assuming Cody used REM because that's what the original code was using. 

We can use px here.

Reply:
Yeah, I just stuck with the format that was already there - I'll switch it to px.

Also, I meant to make a note that I decreased the padding because `15rem` was making the stories in the canvas blocks gigantic in the docs tab.